### PR TITLE
Initial frontend structure

### PR DESCRIPTION
**Pull request by PingvinAustr**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    <!--
+      Notice the use of %PUBLIC_URL% in the tags above.
+      It will be replaced with the URL of the `public` folder during the build.
+      Only files inside the `public` folder can be referenced from the HTML.
+
+      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
+      work correctly both with client-side routing and a non-root public URL.
+      Learn how to configure a non-root public URL by running `npm run build`.
+    -->
+    <title>React App</title>
```
Comment (LanchevychMaxym) [inline public/index.html:27]: Let’s update app name and description

File: public/manifest.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
+{
+  "short_name": "React App",
+  "name": "Create React App Sample",
```
Comment (LanchevychMaxym) [inline public/manifest.json:3]: We can change these names as well
And app logo if we have that one ready

File: public/index.html (hunk 1 above)
Comment (PingvinAustr) [reply public/index.html]: @LanchevychMaxym , app name description is changed now

File: public/manifest.json (hunk 2 above)
Comment (PingvinAustr) [reply public/manifest.json]: @LanchevychMaxym These changes are also applied now. Regarding favicon / logo - we do not have one yet - I will ask PO to provide it on Monday

**Review by EyR1oN — APPROVED** [review]

**Review by Tatiana2424 — APPROVED** [review]

**Review by LanchevychMaxym — APPROVED** [review]

**Pull request merged**